docs/source/gbd2019_models/risk_effects/bmi/index.rst

```diff
@@ -0,0 +1,328 @@
+   * - High Body-Mass Index
+     - Alzheimer's disease and other dementias
+   * - High Body-Mass Index
+     - Chronic kidney disease due to hypertension
+   * - High Body-Mass Index
+     - Chronic kidney disease due to glomerulonephritis
+   * - High Body-Mass Index
+     - Chronic kidney disease due to other and unspecified causes
+   * - High Body-Mass Index
+     - Low back pain
+   * - High Body-Mass Index
+     - Gout
+   * - High Body-Mass Index
+     - Cataract
+   * - High Body-Mass Index
+     - Acute lymphoid leukemia
+   * - High Body-Mass Index
+     - Chronic lymphoid leukemia
+   * - High Body-Mass Index
+     - Acute myeloid leukemia
+   * - High Body-Mass Index
+     - Chronic myeloid leukemia
+   * - High Body-Mass Index
+     - Other leukemia
+   * - High Body-Mass Index
+     - Diabetes mellitus type 2
+   * - High Body-Mass Index
+     - Chronic kidney disease due to diabetes mellitus type 2
+   * - High Body-Mass Index
+     - Burkitt lymphoma
+   * - High Body-Mass Index
+     - Other non-Hodgkin lymphoma
+   * - High Body-Mass Index
+     - Osteoarthritis hip
+   * - High Body-Mass Index
+     - Osteoarthritis knee
+
+[GBD-2019-Capstone-Appendix-BMI2]_
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - Age group start
+     - 9
+     - [20, 24 years)
+   * - Age group end
+     - 235
+     - [95, 125 years)
+
+
+Risk Outcome Pair #1: Ischemic heart disease
+++++++++++++++++++++++++++++++++++++++++++++
+
+:ref:`See ischemic heart disease documentation <2019_cause_ihd>`
+
+The relative risks apply to the incidence rates of acute 
+myocardial infarction and stable angina. They should be 
+applied using the formula: 
+
+incidence(i) = incidence*(1-PAF\ :sub:`r108`\)*RR^{max((BMI_i - TMREL),0)/5} 
+
+The association was evaluated at the cause level, but the 
+associations should be applied to the incidence rates for 
+both nonfatal components of ischemic heart disease. The 
+relative risk for GBD 2019 is for a 5-unit increase in BMI.
+
+PAFs and relative risks can be pulled using the following code: 
+
+rrs = get_draws(gbd_id_type='rei_id', gbd_id=108, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 
+
+pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[108, 493], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 
+
+Risk Outcome Pair #2: Ischemic stroke
++++++++++++++++++++++++++++++++++++++
+
+:ref:`See ischemic stroke documentation <2019_cause_ischemic_stroke>`
+
+The relative risks apply to the incidence rates of acute 
+ischemic stroke. They should be applied using the formula: 
+
+incidence(i) = incidence*(1-PAF\ :sub:`r108`\)*RR^{max((BMI_i - TMREL),0)/5} 
```

Comment:
I think this should apply to arrow 1 and 3 in the IS cause model diagram https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/causes/ischemic_stroke/index.html#cause-model-diagram

Reply:
Thanks! Added